website/src/components/Hero.tsx

```diff
@@ -17,22 +18,26 @@ export function Hero() {
             />
           </div>
           <div className="relative z-10 mx-auto flex w-64 rounded-xl bg-slate-600 shadow-xl md:w-80 lg:w-auto">
-            <Image className="w-full" src={coverImage} alt="" priority />
+            <Image className="w-full" src={coverImage} alt={t('coverImgAlt')} priority />
           </div>
         </div>
         <div className="relative px-4 sm:px-6 lg:col-span-7 lg:pb-14 lg:pl-16 lg:pr-0 xl:pl-20">
           <div className="hidden lg:absolute lg:-top-32 lg:bottom-0 lg:left-[-100vw] lg:right-[-100vw] lg:block lg:bg-slate-100" />
           <h1 className="relative text-4xl font-extrabold text-cyan-900 sm:text-6xl">
             Curator AI
           </h1>
+          <div />
+        </div>
+        <div className='absolute right-2 px-10'>
+          <LanguageSelector />
         </div>
         <div className="bg-white pt-16 lg:col-span-7 lg:bg-transparent lg:pl-16 lg:pt-0 xl:pl-20">
           <div className="mx-auto px-4 sm:px-6 md:max-w-2xl md:px-4 lg:px-0">
             <h2 className="font-display text-4xl font-extrabold text-slate-900 sm:text-6xl">
-              Feeling overwhelmed by all your IT trend sources?
+              {t('hero1')}
             </h2>
             <p className="mt-4 text-3xl text-slate-600">
-              Your personalized trends curator, designed to help you save time.
+              {t('hero2')}
```

Comment:
Rename this translation key as it is not clear:
```suggestion
              {t('hero.content')}
```